start.js

```diff
@@ -86,13 +89,14 @@ function runFastify (args, cb) {
 
   loadModules(opts)
 
-  var file = null
-  try {
-    file = require(path.resolve(process.cwd(), opts._[0]))
-  } catch (e) {
-    return module.exports.stop(e)
+  const filePath = path.resolve(process.cwd(), opts._[0])
+
+  if (!fs.existsSync(filePath)) {
+    return module.exports.stop('', `${opts._[0]} doesn't exist within ${process.cwd()}`)
   }
 
+  const file = require(filePath)
```

Comment:
```suggestion
  let file = null
  try {
    file = require(filePath)
  } catch (e) {
    return module.exports.stop(e)
  }
```

Reply:
You just ensure the `filePath` exists in folder, how about the require files in `filePath`? So you need to add this try catch to ensure `require(filePath)` run correctly.